src/Authorization.AspNetCore/AuthorizationValidationRule.cs

```diff
@@ -28,6 +29,8 @@ public Task<INodeVisitor> ValidateAsync(ValidationContext context)
         {
             return Task.FromResult((INodeVisitor)new EnterLeaveListener(_ =>
             {
+                AuthorizeAsync(null, context.Schema as IProvideMetadata, context, null).GetAwaiter().GetResult();
```

Comment:
The main change in this PR.

Reply:
I can remove cast after merging https://github.com/graphql-dotnet/graphql-dotnet/pull/2036

Reply:
done